build/templates/examples.rst.mako

```diff
@@ -27,13 +27,23 @@
 
     with open(f'./src/{module_name}/LATEST_RELEASE') as vf:
         latest_release_version = vf.read().strip()
-    released_zip_url = 'https://github.com/ni/nimi-python/releases/download/{}/{}_examples.zip'.format(latest_release_version, module_name)
-
-    example_url_base = 'https://github.com/ni/nimi-python/blob/'
 
     from packaging.version import Version
     v = Version(module_version)
 
+    # Check if the module name and version match the old tag formatting criteria
+    use_old_tag_format = (
+        module_name in ['nidcpower', 'nidigital', 'nidmm', 'nifake', 'niswitch', 'nimodinst', 'nifgen', 'niscope', 'nise', 'nitclk']
+        and v.major <= 1 and v.minor <= 4 and v.micro <= 9
```

Comment:
```suggestion
        and latest_release_version == '1.4.9'
```

`tox -e codegen` shows that what we have is resulting in unwanted changes to the examples.rsts and we really don't need to worry about the old format for previous versions or versions past 1.4.9.